lib/charms/operator_libs_linux/v0/snap.py

```diff
@@ -260,24 +267,51 @@ def unset(self, key) -> str:
         """
         return self._snap("unset", [key])
 
-    def _install(self, channel: Optional[str] = "") -> None:
+    def _install(self, channel: Optional[str] = "", cohort: Optional[str] = "") -> None:
         """Add a snap to the system.
 
         Args:
+          confinemenet: strict or classic
           channel: the channel to install from
+          cohort: optional, the key of a cohort that this snap belongs to
         """
         confinement = "--classic" if self._confinement == "classic" else ""
```

Comment:
Ah, related to above, it seems we're just using the `self._confinement` so I guess that argument description can just go?